src/plugin.js

```diff
@@ -168,11 +169,21 @@ const initPlugin = function(player, options) {
         if (player.ended()) {
           return resetMonitor();
         }
+
       }
 
       fn.call(this);
     };
 
+    if (videojs.browser.IS_IOS) {
+      player.on(['stalled', 'suspend'], (e) => {
+        if (window.navigator.onLine && player.bufferedEnd() > player.currentTime() + 1) {
```

Comment:
Let's try to further minimize potential for false positives by adding `!player.paused()` here.

The question came up: is the magic `1` needed? Should it be another number like [this](https://github.com/videojs/http-streaming/blob/446cb101dd4acca2dc9d2d56fb97b18ca31d1d3a/src/ranges.js#L17).